<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="2">
     <widget class="QLineEdit" name="xSize"/>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="specBrowseButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QLineEdit" name="inImage"/>
    </item>
    <item row="4" column="1" colspan="4">
     <widget class="QLineEdit" name="inSave">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="start">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="imageBrowseButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLineEdit" name="ySize"/>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>  Cutout size in arcsec:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>  Save cutouts at:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>  Path to Image:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLineEdit" name="inSpectra"/>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QTextBrowser" name="infoBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 203);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; text-decoration: underline;&quot;&gt;NIRSpec Cutout Tool&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Info:&lt;/span&gt; This is the cutout tool for MOSViz. This tool will use the information in the NIRSpec files to make cutouts. The user must specify the path to the field image and a directory containing the NIRSpec spectra. The spectra file should follow the NIRSpec naming scheme as follows: &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;programName_objectName_instrument_filter_ grating_(s2d|x1d).fits &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The user can set &amp;quot;cutout size&amp;quot; which will determine the height and width of the target centered cutouts. &lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note: &lt;/span&gt;The folder containing the cutouts will be saved in the same directory as the spectra. The user may change the save path but this would make the generated MOSViz table unique to this computer (you will not be able to share it). &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>  Path to spectra directory:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="savePathButton">
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>  Progress:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
